### DISCUSSION TITLE
Problem with VPC generation

### DISCUSSION THREAD

**Discussion by alvarogg19** [LAPKB/Pmetrics#211] [Q&A]
Hello, I have a problem generating VPC. 

When I have my PM_result, I load it and use the next code:

> Res$validate(limits = c(0, 3))
Covariates in your data file:  icu, male, age, wt, ht, alb, creat, mv, cg, ckdepi, ckdeas, mdrd, mdrdjp, mdrdch
Enter any covariates to be binned, separated by commas (<Return> for none): ckdepi
Now optimizing clusters for dose/covariates.
First step is a Gaussian mixture model analysis, followed by an elbow plot.
Press <Return> to start cluster analysis for dose, ckdepi: 
Now performing Gaussian mixture model analysis.fitting ...
  |======================================================================| 100%
Most likely number of clusters is 1.
Press <Return> to see classification plot: 
Press <Return> to see elbow plot: 
Error in do_one(nmeth) : 
  NA/NaN/Inf en llamada a una función externa (arg 1)
Además: Warning message:
In storage.mode(x) <- "double" : NAs introducidos por coerción

You see, when it gets to the point of generating the elbow plot, it breaks. With the example file I have been able to generate the VPCs correctly. Anyone knows how can I fix it?

Thank you so much, best wishes,

Alvaro

**Comment by mnneely** [comment]
It seems that the ckdepi covariate is not binning very well - only one cluster being made. 1) Is that covariate appearing in the equations in your model? 2) What are the range of values for the covariate in your data? 3) Doses and covariates are binned together, so do you have a range of doses in the data?

The upshot is that your dose/covariates are only coming up in one bin, which suggests they are all the same.
